scripts/compare_noupdate_xml_records.py

```diff
@@ -105,8 +110,20 @@ def process_data_node(data_node):
                 tree = etree.parse(os.path.join(addon_dir, *xml_path))
             except etree.XMLSyntaxError:
                 continue
-            for data_node in tree.xpath("/openerp/data"):
-                process_data_node(data_node)
+            # FIX by Alex: Make it compatible between the 9.0 and 10.0 version;
+            if version == 'old':
+                for data_node in tree.xpath("/openerp/data"):
+                    process_data_node(data_node)
+            elif version == 'new':
```

Comment:
This doesn't totally serve, as you can have in 9.0 both `<odoo>` or `<openerp>` tags.